analysis/deep2_r23_o32/general.py

```diff
@@ -300,12 +302,12 @@ def run_grid_r23_o32_analysis(dataset, y_correction, n_split, adaptive=False, du
     # R_temp_calcul
     # Not going to run the R_temp_calcul.run_function for the 'bin_valid' table because these values
     # are proven to be incomplete. The bin_valid_rev table is different.
-    R_temp_calcul.run_function(fitspath, dataset, verification_table_revised, dustatt=False)
+    r_temp_calcul.run_function(fitspath, dataset, verification_table_revised, dustatt=False)
```

Comment:
We really want to use MSC for common functionality.

Reply:
Can you clarify a little? This file used to calculate R, temp, and metallicity, but now it calls on MSC to make those calculations.